From 014dc43447fa553e875984ac4a528e5916cc9e06 Mon Sep 17 00:00:00 2001
From: Carlos Santos <unixmania@gmail.com>
Date: Sat, 29 Feb 2020 22:41:30 -0300
Subject: [PATCH] runlevels: do not add agetty.tty[1-6] if MKSYSVINIT=yes

Buildroot starts a single getty, according to the system configuration.
Also tty[1-6] may not exist (e.g. embedded devices with serial consoles
only).

Signed-off-by: Carlos Santos <unixmania@gmail.com>
---
 runlevels/Makefile | 6 ------
 1 file changed, 6 deletions(-)

diff --git a/runlevels/Makefile b/runlevels/Makefile
index 73843d8b..dbfb59ca 100644
--- a/runlevels/Makefile
+++ b/runlevels/Makefile
@@ -90,12 +90,6 @@ install:
 			fi; \
 			ln -snf ${INITDIR}/"$$x" ${SHUTDOWNDIR}/"$$x" || exit $$?; done \
 	fi
-	if test "${MKSYSVINIT}" = yes && test "${OS}" = Linux; then \
-		for x in tty1 tty2 tty3 tty4 tty5 tty6; do \
-			ln -snf ${INITDIR}/agetty ${DESTDIR}/${INITDIR}/"agetty.$$x" || exit $$?; \
-			ln -snf ${INITDIR}/agetty.$$x ${DEFAULTDIR}/"agetty.$$x" || exit $$?; \
-		done; \
-	fi
 
 check test::
 
-- 
2.18.2

